test/cbmc/proofs/Sntp_SendTimeRequest/README.md

```diff
@@ -0,0 +1,20 @@
+Sntp_SendTimeRequest proof
+==============
+
+This directory contains a memory safety proof for Sntp_SendTimeRequest.
+
+To run the proof.
+-------------
+
+* Add `cbmc`, `goto-cc`, `goto-instrument`, `goto-analyzer`, and `cbmc-viewer`
+  to your path.
+* Run `make`.
+* Open html/index.html in a web browser.
+
+To use [`arpa`](https://github.com/awslabs/aws-proof-build-assistant) to simplify writing Makefiles.
```

Comment:
I think it would be okay to delete from here on in the README, I don't think you are using arpa.